src/bp/common/validation.ts

```diff
@@ -24,7 +25,8 @@ export const BotEditSchema = Joi.object().keys({
     .min(3)
     .max(50)
     .required(),
-  category: Joi.string(),
+  // tslint:disable-next-line:no-null-keyword
+  category: Joi.string().allow(null),
```

Comment:
null value

Reply:
Just so you know bot without category is set to null at the moment